src/NuGet.Services.AzureSearch/Db2AzureSearch/NewPackageRegistrationProducer.cs

```diff
@@ -215,6 +217,24 @@ private async Task<IReadOnlyList<PackageRegistrationInfo>> GetPackageRegistratio
             }
         }
 
+        private bool ShouldSkipPackageRegistration(PackageRegistration packageRegistration)
+        {
+            if (_options.Value.SkipPackagePrefixes == null)
+            {
+                return false;
+            }
+
+            foreach (var skippedPackagePrefix in _options.Value.SkipPackagePrefixes)
```

Comment:
I think this and following lines can be replaced with something like
```
return _options.Value.SkipPackagePrefixes.Any(prefix => packageRegistration.Id.StartsWith(prefix, StringComparison.OrdinalIgnoreCase));
```

Reply:
I don't have a strong preference either ways here. LINQ is shorter but results in a longer line here.